myst_nb/parser.py

```diff
@@ -112,6 +113,33 @@ def parse_block(src, start_line):
                 dup_ref["fixed"] = True
         return tokens
 
+    def parse_code_cell(cell, start_line):
+        tokens = [
+            Token(
+                "nb_code_cell",
+                "",
+                0,
+                meta={"cell": cell},
+                map=[start_line, start_line],
+            )
+        ]
+        for i, output in enumerate(cell["outputs"]):
```

Comment:
could you add some comments for what this section is doing (e.g. "If outputs contain markdown, turn it into a new markdown cell to be parsed")